workflow/automation/lib/MgmtDB.py

```diff
@@ -367,8 +371,8 @@ def add_retries(self, n_max_retries: int):
                     "AND status <= (SELECT id FROM status_enum WHERE state = 'completed') ",
                     (run_name, proc_type),
```

Comment:
Could be made more efficient (and clearer) with a SQL JOIN:
```sql
SELECT COUNT(*)
FROM state s
JOIN status_enum se ON s.status <= se.id
WHERE se.state = 'completed';
```

Reply:
double check that does the same thing (I ran it in the SQL DB engine of "my head"), but I think it does.

Reply:
can be simply `status <= 5`.

Reply:
oh I didn't see that wasn't your code. Ignore this then.